### PR TITLE
[docs] Named Vectors Examples

### PR DESCRIPTION
**Pull request by davidmyriel**
[PREVIEW](https://deploy-preview-1156--condescending-goldwasser-91acf0.netlify.app/documentation/concepts/vectors/#named-vectors)

### CODE REVIEW

**Comment by Anush008** [comment]
We're not adding the SDK snippets?

File: qdrant-landing/content/documentation/concepts/vectors.md [hunk 1]
```diff
@@ -924,13 +924,15 @@ client.Query(context.Background(), &qdrant.QueryPoints{
...
-Aside from storing multiple vectors of the same shape in a single point, Qdrant supports storing multiple different vectors in a single point.
+In Qdrant, you can store multiple vectors of different sizes in the same data [point](/documentation/concepts/points/). This is useful when you need to define your data with multiple embeddings to represent different features or modalities (e.g., image, text or video). 
 
-Each of these vectors should have a unique configuration and should be addressed by a unique name.
-Also, each vector can be of a different type and be generated by a different embedding model.
+To store different vectors from each point, you need to create separate named vector spaces in the [collection](/documentation/concepts/collections/). You can define these vector spaces during collection creation and manage them independently.
 
-To create a collection with named vectors, you need to specify a configuration for each vector:
+<aside role="status">
+Each vectors should have a unique name. Vectors can represent different modalities and you can use different embedding models to generate them.
```
Comment (Anush008) [inline qdrant-landing/content/documentation/concepts/vectors.md:932]:
```suggestion
Each vector should have a unique name. Vectors can represent different modalities and you can use different embedding models to generate them.
```

File: qdrant-landing/content/documentation/concepts/vectors.md [hunk 2]
```diff
@@ -924,13 +924,15 @@ client.Query(context.Background(), &qdrant.QueryPoints{
 
 ## Named Vectors
 
-Aside from storing multiple vectors of the same shape in a single point, Qdrant supports storing multiple different vectors in a single point.
+In Qdrant, you can store multiple vectors of different sizes in the same data [point](/documentation/concepts/points/). This is useful when you need to define your data with multiple embeddings to represent different features or modalities (e.g., image, text or video). 
 
-Each of these vectors should have a unique configuration and should be addressed by a unique name.
-Also, each vector can be of a different type and be generated by a different embedding model.
+To store different vectors from each point, you need to create separate named vector spaces in the [collection](/documentation/concepts/collections/). You can define these vector spaces during collection creation and manage them independently.
```
Comment (Anush008) [inline qdrant-landing/content/documentation/concepts/vectors.md:929]:
```suggestion
To store different vectors for each point, you need to create separate named vector spaces in the [collection](/documentation/concepts/collections/). You can define these vector spaces during collection creation and manage them independently.
```

**Comment by davidmyriel** [comment]
> We're not adding the SDK snippets?

I threw this together in 5 min. Can you commit the snippets? I may make a mistake.

**Review by Anush008 — APPROVED** [review]

**Pull request merged**